.github/workflows/build.yml

```diff
@@ -0,0 +1,63 @@
+name: Spidermonkey
+
+on:
+  push:
+    branches: [ wip-esr91 ]
```

Comment:
Should we name this `next` like it's described in https://github.com/mozilla-spidermonkey/spidermonkey-embedding-examples#spidermonkey-embedding-resources ? (In case the ESR target changes during one ESR period, as it did with esr59 to esr60)

Reply:
As I already said on matrix, the `next` branch will make things easier, as on ESRs just create branch from `next`, with minor changes.